### DISCUSSION TITLE
max7219.c

### DISCUSSION THREAD

**Discussion by AchimPieters** [UncleRus/esp-idf-lib#289] [Q&A]
@UncleRus how do I set 3 units inline? So it will say AWAY in the display? 

```
static const uint64_t AWAY[] = {
                0x0000002969af2926,
                0x0000004a4b7a4a32,
                0x0000000202020505
        };
```

I only see the first line _0x0000002969af2926_ ?

**Comment by trombik** [comment]
moved from Issues because it's a question.
